Clinical-Genomics/scout
# Hide dismissed variants in variants page and export variants

Adds a checkbox to hide/show dismissed variants in variants page (snvs, svs, cancer snvs, cancer svs):


https://user-images.githubusercontent.com/28093618/102990082-11cacd80-4517-11eb-9fec-c6fae6c0d267.mov


fix #2194 

**How to test**:
1. Test locally or on stage on at least a rare disease and and a cancer case
2. Go to case variants: snvs, svs, cancer snvs, cancer svs(this type of variants is better tested on the local demo cancer case) and dismiss one or more variants.
3. Check and uncheck the "Hide dismissed" checkbox and make sure that it works
4. Make sure that if the checkbox is checked then the dismissed variants are not exported using the export variants button from the filters form

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
